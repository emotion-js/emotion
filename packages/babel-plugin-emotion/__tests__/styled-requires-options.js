// @flow
import { createInlineTests } from 'old-babel-tester'

const cases = {
  'hash generation no file system': {
    code: 'import styled from "@emotion/styled";\nstyled.h1`color:blue;`',
    filename: undefined
  }
}

createInlineTests('styled inline', cases)
