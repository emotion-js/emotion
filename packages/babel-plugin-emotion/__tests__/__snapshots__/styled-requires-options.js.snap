// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`styled inline babel 6 hash generation no file system 1`] = `
"import _styled from \\"@emotion/styled-base\\";

_styled(\\"h1\\", {
  target: \\"e1vqloan0\\"
})({
  name: \\"14ksm7b\\",
  styles: \\"color:blue;\\"
});"
`;

exports[`styled inline babel 7 hash generation no file system 1`] = `
"import _styled from \\"@emotion/styled-base\\";

_styled(\\"h1\\", {
  target: \\"e1vqloan0\\"
})({
  name: \\"14ksm7b\\",
  styles: \\"color:blue;\\"
});"
`;
